test/jest_integration/runIdpServer.js

```diff
@@ -0,0 +1,8 @@
+const { runServer } = require('saml-idp');
+
+// Create certificate pair on the fly and pass it to runServer
+runServer({
+  acsUrl: 'http://localhost:5601/_opendistro/_security/saml/acs',
```

Comment:
is it possible to run idp server with acs `/_plugins/_security/saml/acs`? or will it be covered in next iteration of this PR?

Reply:
did not understand, can you please explain?